src/Spice86.Core/Emulator/InterruptHandlers/Input/Mouse/MouseDriver.cs

```diff
@@ -10,12 +10,25 @@ namespace Spice86.Core.Emulator.InterruptHandlers.Input.Mouse;
 using Spice86.Core.Emulator.InterruptHandlers.VGA.Records;
 using Spice86.Core.Emulator.Memory.Indexable;
 using Spice86.Shared.Emulator.Memory;
+using Spice86.Shared.Emulator.Mouse;
 using Spice86.Shared.Interfaces;
 
+using System;
+
 /// <summary>
 ///     Driver for the mouse.
 /// </summary>
 public class MouseDriver : IMouseDriver {
+    private int _leftButtonPressCount;
```

Comment:
Make a class to regroup all the information about a button? You could then have a dictionary to map a button to an instance, would make code shorter

Reply:
En effet, j'ai mis en place un dictionnaire. :)